File: cpp_e9zMZ/cpp/07-simdienthoai/88.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    string phone = "0764.555.578";
    string result = "No";

    phone = myReplace(phone, ".");
    for (int i = 1; i < phone.length() - 3; i++) {
        char current = phone[i];
        if (current == phone[i + 1] && current == phone[i + 2] && current == phone[i + 3]) {
            result = "Yes";
            break;
        }
    }
    cout << result << endl;
}

File: python_ToSXs/python/07-simdienthoai/88.py
phone = "0764.555.578"

phone = phone.replace(".", "")
result = "No"

for i in range(1, len(phone) - 3):
    if (phone[i] == phone[i + 1] and
            phone[i] == phone[i + 2] and
            phone[i] == phone[i + 3]):
        result = "Yes"
        break

print(result)
